notebooks/NIRSPEC/FSlit/JWPipeNB-NIRSpec-FS.ipynb

```diff
@@ -869,6 +870,28 @@
     "det1dict['jump']['maximum_cores'] = 'half'"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "6b351726",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "The `refpix_algorithm` parameter in the `refpix` step is applicable only to NIR data. For NIRSpec full-frame data using traditional readout modes, the default algorithm—set in the parameter reference file as of Build 12.0—is 'sirs' (Simple Improved Reference Subtraction), which uses reference pixels from the left and right sides. For subarray data or non-traditional readouts, the algorithm automatically defaults to 'median'.\n",
+    "\n",
```

Comment:
Can you be more specific than 'traditional' readout modes?  I.e., I think you mean that NRS, NRSRAPID, etc. steps use the SIRS algorithm, but NRSIRS2 doesn't because that method is essentially already applied on-board as part of the readout electronics.